README.md

```diff
@@ -7,9 +7,9 @@ Binding ([JSR-367](https://jcp.org/en/jsr/detail?id=367)).
 
 # <a name="Latest_News"></a>Latest News
 
-## December 28, 2018 - Yasson 1.0.3 Final Release ##
+## September 06, 2019 - Yasson 1.0.5 Final Release ##
```

Comment:
I think we can remove this section because we already have the maven central badge that automatically shows the latest release version. I'd like to have as few places as possible to manually update after each release

Reply:
Well I would not do that. It is the title which should tell the visitor that we have released new version. -> https://eclipse-ee4j.github.io/yasson/

Reply:
I don't think Yasson is a large/active enough project to warrant a "news" section, it will pretty much just be a list of latest releases right?